Assets/LeapMotion/Scripts/HandRepresentation.cs

```diff
@@ -14,12 +14,15 @@ public abstract class HandRepresentation
     public Chirality RepChirality { get; protected set;}
     public ModelType RepType { get; protected set;}
     public Hand MostRecentHand { get; protected set; }
+    public Hand PostProcessHand { get; set; }
+    public HandPool.ModelGroup Group { get; set; }
```

Comment:
We talked about this in the issue about this abstraction, but just re-iterating here.  HandRepresentation is part of the underlying abstraction, and HandPool is an implementation of that abstraction, thus HandRepresentation is not technically allowed to know about HandPool or ModelGroup.

However, since this has been a hassle for a long while, we should resolve the abstraction decision first, and then this should become more straightforward.